### ISSUE TITLE
Oppdrag

### ISSUE THREAD

**Issue by LarsFredrikHolstTry** [LarsFredrikHolstTry/bawser#22]
Forslag til oppdrag kan vi legge inn her, tenker de første 5-10 oppdragene bør gå ganske kjapt og hjelpe brukeren i gang med spillet.
Tenker også at kombinerte oppdrag er ganske fet, men det er bare min personlige mening (10 krim, 5 biltyveri og 5 brekk er 1 oppdrag)

Vi burde hvertfall ha rundt 50 oppdrag klare til lanseringsdagen.

**Comment by Cobrazking** [comment]
Ja kombinasjonsoppdrag er stilig :) 

Tenker du at vi skal lage en slags story? (så man kan utføre forskjellige stories?)  Eller skal oppdragene baseres på kun ingame funksjoner uten story?

**Comment by LarsFredrikHolstTry** [comment]
Stories hadde vært rått, så kan spillerne velge selv hvilken story de vil på eller?
Liker også tidsbaserte oppdrag slik du har laget på skissen her:
![oppdrag](https://user-images.githubusercontent.com/29153280/209880313-b2c9cb25-39ae-460d-9558-99ece6ac7646.png)

**Comment by Cobrazking** [comment]
Ja, kunne vært 1 eller 2 stories i starten man kan velge hvem man starter på. Så kan vi supplere med flere stories etterhvert. Noen stories kan jo også eventuelt være betalt innhold også på sikt (bare en tanke)

**Comment by Cobrazking** [comment]
![MALNY2022-oppdrag](https://user-images.githubusercontent.com/120334498/209964696-2058a3df-738e-4bb5-99ed-017c7e87bc2c.png)
Første utkast av eksempel på oppsett
